__tests__/games.test.js

```diff
@@ -0,0 +1,42 @@
+const app = require("../app.js")
+const request = require("supertest")
+const testData = require("../db/data/test-data")
+const db = require("../db/connection")
+const seed = require("../db/seeds/seed")
+const categories = require("../db/data/test-data/categories.js")
```

Comment:
This import is unnecessary as the testData handles all the data we need. It may also be causing your test confusion whenever you refer to categories